<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LanguageWidget</class>
 <widget class="QWidget" name="LanguageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#languageTitle {
	qproperty-alignment: AlignCenter;
	font-size: 24px;
	color: #58595b;
}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>771</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="languageContainer"/>
  </widget>
  <widget class="QLabel" name="languageTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#language_message_dialog_title {
	qproperty-alignment: AlignCenter;
	font-size: 24px;
	color: #58595b;
}</string>
   </property>
   <property name="text">
    <string>Choose your language</string>
   </property>
  </widget>
  <widget class="QLabel" name="softwareVersions">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>500</y>
     <width>731</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#softwareVersions {
	color: #50666666;
	font-family: Muli-SemiBold;
	font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
